src/strtab.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> Index<usize> for Strtab<'a> {
     /// **NB**: this will panic if the underlying bytes are not valid utf8, or the offset is invalid
     #[inline(always)]
     fn index(&self, offset: usize) -> &Self::Output {
-        get_str(offset, &self.bytes, self.delim).unwrap()
+        self.get(offset).unwrap().unwrap()
```

Comment:
Ahh, [Travis is monkeying with their Mac infrastructure](https://www.traviscistatus.com/incidents/kdvnlc4w3n67) as part of an upgrade. Today's build delays are not normal.